ParcialProgramacion4/src/PuestoJerarquico.java

```diff
@@ -0,0 +1,7 @@
+public class PuestoJerarquico extends Puesto{
+    private static int aniosMinimosParaCambiar;
```

Comment:
prefiero que volvamos a la prehistoria donde se escribia año como anno
```suggestion
    private static int annosMinimosParaCambiar;
```